### ISSUE TITLE
Embed Grafana Dashboards In RunMe Notebooks

### ISSUE THREAD

**Issue by jlewi** [stateful/vscode-runme#1361]
I think one of the key value propositions of RunMe is the ability to combine rich visualizations of observability data with, NL, and executable commands. [Datadog notebooks](https://docs.datadoghq.com/notebooks/) are a great illustration of the UX for observability. RunMe has a huge advantage over that in that it can execute commands.

I'm wondering if supporting embedding Grafana dashaboards would be a way to quickly bring some of Grafana's rich visualization capabilities to RunMe?

According to this [post](https://grafana.com/blog/2023/10/10/how-to-embed-grafana-dashboards-into-web-applications/)  possible to embed grafana dashboards into web applications. I think there's a caveat that dashboards have to be public; i.e. they can't be embedded with Grafana Authentication.

If Grafana is just embedded as an iframe; does that make it super easy to support that in RunMe.Dev? Perhaps its even already supported?

Related issue: grafana/grafana#82586

**Comment by sourishkrout** [comment]
Yes, @jlewi! This is something near the the top of our list when it comes to Cloud Resource renderers.

It's not already supported but based on what we know about Grafana should be lower hanging fruit (within the constraints of Grafana's licensing). We actually have a integration blog post about the same topic on our blog: https://stateful.com/blog/grafana-in-react.

We're planning to launch the first set of Cloud Resource renderers (GKE & GKE and EC2 & EKS) in about early June and subsequently are planning to go broad on Cloud Service Provider's resources before going deep on any particular provider.
